ui/main/src/app/modules/card/components/card-body/card-body.component.ts

```diff
@@ -233,6 +236,18 @@ export class CardBodyComponent implements OnChanges, OnInit, OnDestroy {
         }
     }
 
+    private computeUserAllowedToEdit() {
+        this.isUserAllowedToEdit =
+            this.cardState.editCardEnabledOnUserInterface && this.doesTheUserHavePermissionToEditCard();
```

Comment:
The editCardEnabledOnUserInterface shall not be checked anymore as even if the edit link is not visible on screen it should be possible to edit the card via js code from the template